plugins/orchestrator-backend/src/service/router.ts

```diff
@@ -876,7 +876,7 @@ function setupInternalRoutes(
     'abortWorkflow',
     async (c, _req, res, next) => {
       const instanceId = c.request.params.instanceId as string;
-      const endpointName = 'getInstanceById';
+      const endpointName = 'abortWorkflow';
       const endpoint = `/v2/instances/${instanceId}/abort`;
```

Comment:
/v2/instances here also looks wrong @batzionb

Reply:
https://github.com/janus-idp/backstage-plugins/pull/2141

Reply:
@gciavarrini

Reply:
@rgolangh i wouldn't say "wrong" but inconsistent. Thanks for the fix :)